template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Module-VisualBasic/.template.config/localize/templatestrings.en.json

```diff
@@ -0,0 +1,5 @@
+{
+  "author": "Microsoft",
+  "name": "Module",
+  "description": "Creates a new Visual Basic module"
```

Comment:
Do we want to add/replace the original item template descriptions?

![image](https://user-images.githubusercontent.com/9663150/214923446-babea463-dc32-46c1-b1ee-a647fdb95093.png)

(I don't have a strong opinion here, I just noticed.)

Also, in other item templates - are we ending the description with a period character?

Reply:
From what I can see we don't follow a stable pattern.
@baronfel , should we unite the approach (description with/without period character)?